plugin/input/file/watcher.go

```diff
@@ -18,6 +20,7 @@ type watcher struct {
 	watcherCh         chan notify.EventInfo
 	shouldWatchWrites bool
 	logger            *zap.SugaredLogger
+	fdCounter         atomic.Int64
```

Comment:
where is it used?

Reply:
it's artifacts, thx for notice. Removed.